backend/src/main/java/com/google/sps/servlets/AddParticipantServlet.java

```diff
@@ -54,10 +56,39 @@
   private static final String PROPERTY_FIRSTPARTICIPANT = "firstParticipant";
   private static final String PROPERTY_SECONDPARTICIPANT = "secondParticipant";
 
+  // HTTP Request JSON key constants
+  private static final String JSON_TIMEAVAILABLEUNTIL = "timeAvailableUntil";
```

Comment:
I would prefix these with REQUEST_ and optionally remove the JSON part as well. The pertinent information is that these are keys into the request.

Reply:
Thanks for the suggestion, done!